Assets/Editor/UnitTests/Models/SchedulerEditorTest.cs

```diff
@@ -38,20 +38,18 @@ public void Init()
         if (FunctionsManager.ScheduledEvent == null)
         {
             new FunctionsManager();
-            FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
         }
 
+        FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
+
         if (PrototypeManager.SchedulerEvent == null)
         {
             new PrototypeManager();
```

Comment:
We can totally remove the if check though.

Yes I agree with you. Static things caused issues already. I am still thinking of a global static class that holds all the current static classes. Just to make them easy to access.